ashlar/migrations/0001_initial.py

```diff
@@ -20,7 +20,7 @@ class Migration(migrations.Migration):
                 ('created', models.DateTimeField(auto_now_add=True)),
                 ('modified', models.DateTimeField(auto_now=True)),
                 ('version', models.IntegerField()),
-                ('schema', django_pgjson.fields.JsonBField()),
+                ('schema', djsonb.fields.JsonBField()),
```

Comment:
Is it feasible to create a new migration for this rather than editing old migrations? That has caused problems in the past because it requires a database drop / reprovision.